README.txt
TEQneers Search Engine Optimization

This extension provides an indexed google/xml-sitemap, enhanced metatag-support and pagetitel-manipulations for TYPO3.
It's an replacement of the "metatag"-extension.

Support:    http://forge.typo3.org/projects/extension-tq_seo/
Bugtracker: http://forge.typo3.org/projects/extension-tq_seo/issues
Homepage:   http://opensource.teqneers.com/typo3/tq-seo.html

ChangeLog
2014-03-27 Markus Blaschke, TEQneers GmbH & Co. KG  <blaschke@teqneers.de>
	* Added metatag editor (currently for opengraph)

2013-10-04 Markus Blaschke, TEQneers GmbH & Co. KG  <blaschke@teqneers.de>
	* [!!!] Requires TYPO3 6.0
	* Refactored tq_seo for TYPO3 6.0 (namespaces and better xclass support)
	* Added update script

2012-12-05 Markus Blaschke, TEQneers GmbH & Co. KG  <blaschke@teqneers.de>
	* Fixed TYPO3 6.0 issue

2012-09-13 Markus Blaschke, TEQneers GmbH & Co. KG  <blaschke@teqneers.de>
	* Fixed typo in manual

2012-06-07 Markus Blaschke, TEQneers GmbH & Co. KG  <blaschke@teqneers.de>
	* Added geo positioning into pages meta tags
	* Added link to sitemal.xml index page into sitemap backend module
	* Added tq_seo class for external metatag manipulations for 3rd party extensions
	* Added better caching of sql queries (sitemap link indexer)

2012-05-25 Markus Blaschke, TEQneers GmbH & Co. KG  <blaschke@teqneers.de>
	* Release of 5.0.0
	* Added SEO backend page-module
	* Fixed severy bugs/issues
	* Added several improvements
	* Fixed some Simulate Static URLs issues

2012-02-03 Markus Blaschke, TEQneers GmbH & Co. KG  <blaschke@teqneers.de>
	* Release of 4.0.1
	* Fixed typo in ext_emconf

2011-12-22 Markus Blaschke, TEQneers GmbH & Co. KG  <blaschke@teqneers.de>
	* Added browser title (Thanks to Jochen Weiland)

2011-12-22 Markus Blaschke, TEQneers GmbH & Co. KG  <blaschke@teqneers.de>
	* Fixed tt_news support (Thanks to Tobias Arendt)
	* Fixed TYPO3 4.6 support  (still need fix for http://forge.typo3.org/issues/31333)

2011-10-28 Markus Blaschke, TEQneers GmbH & Co. KG  <blaschke@teqneers.de>
	* Release of 4.0

2011-10-14 Markus Blaschke, TEQneers GmbH & Co. KG  <blaschke@teqneers.de>
	* Implemented template code for Google Analytics/Piwik as Fluid template (TYPO3 4.5 is required now but it's awesome)
	* Implement asynchronous Google Analytics/Piwik
	* Implemented Mozilla's DoNotTrack with Piwik

2011-09-30 Markus Blaschke, TEQneers GmbH & Co. KG  <blaschke@teqneers.de>
	* Added template code for google analytics and piwik to TypoScript setup (easy to customize now)

2011-09-13 Markus Blaschke, TEQneers GmbH & Co. KG  <blaschke@teqneers.de>
	* Implemented sitemap-listing manipulations
	* Implemented better control center

2011-09-06 Markus Blaschke, TEQneers GmbH & Co. KG  <blaschke@teqneers.de>
	* Implemented backend modules (Bug #28498)
	* Implemented robots.txt support (Bug #28195)

2011-09-01 Markus Blaschke, TEQneers GmbH & Co. KG  <blaschke@teqneers.de>
	* Added tt_news support (auto-metatags collecting)

2011-08-30 Markus Blaschke, TEQneers GmbH & Co. KG  <blaschke@teqneers.de>
	* Fixed #29347
	* Fixed sitemap output (duplicate sql queries and missing limitToCurrentLanguage)

2011-07-21 Markus Blaschke, TEQneers GmbH & Co. KG  <blaschke@teqneers.de>
	* Splitted sitemap building process: builder and output

2011-07-13 Markus Blaschke, TEQneers GmbH & Co. KG  <blaschke@teqneers.de>
	* Added hooks
	* Fixed metatag-urls (full url now)
	* Added typolink sitemap-parser
	* Updated some code-documentation
	* Reduced sql statements
	* Removed deprecated stuff
	* Clear cache possibility is now disabled by default, you should use the scheduler task instead
	* Fixed some other minor issues, bugs, stuff ...

2011-07-01 Markus Blaschke, TEQneers GmbH & Co. KG  <blaschke@teqneers.de>
	* Version 2.5.3
	* Fixed sitemap index page output (wrong page number) in very big TYPO3 installations

2011-06-29 Markus Blaschke, TEQneers GmbH & Co. KG  <blaschke@teqneers.de>
	* Version 2.5.2
	* Fixed missing definition of tx_tqseo_sitemap in ext_tables.sql (Bug #27658, thanks to Dmitri Pisarev)

2011-06-21 Markus Blaschke, TEQneers GmbH & Co. KG  <blaschke@teqneers.de>
	* Version 2.5.1
	* Fixed again the length of some fields (Thanks to Stéphane Perrenoud)

2011-05-10 Markus Blaschke, TEQneers GmbH & Co. KG  <blaschke@teqneers.de>
	* Implemented basic settings table (disabled by now)
	* Improved documentation (added suggested extensions, fixed style - again)

2011-05-05 Markus Blaschke, TEQneers GmbH & Co. KG  <blaschke@teqneers.de>
	* Fixed xclass support

2011-05-04 Markus Blaschke, TEQneers GmbH & Co. KG  <blaschke@teqneers.de>
	* Version 2.5.0
	* Moved sitemap configuration from the ext-conf to constants, ext-conf now flaged as deprecated
	* Updated documentation

2011-04-26 Markus Blaschke, TEQneers GmbH & Co. KG  <blaschke@teqneers.de>
	* Fixed canonical tag 50 chars limit bug  (#18095 in bugs.typo3.org)

2011-04-07 Markus Blaschke, TEQneers GmbH & Co. KG  <blaschke@teqneers.de>
	* Updated documentation

2011-04-06 Markus Blaschke, TEQneers GmbH & Co. KG  <blaschke@teqneers.de>
	* Version 2.4.2
	* Disabled sourceopt (if installed) in sitemap pages (Thanks to Robert Wunsch)
	* Fixed db-issue with integers (Thanks to Robert Wunsch, again)

2011-02-11 Markus Blaschke, TEQneers GmbH & Co. KG  <blaschke@teqneers.de>
	* Disabled by default services (google analytics/piwik) output if config.disableAllHeaderCode is active to prevent double hit-logging (eg. for ajax-calls, can be renabled by constants/typoscript)

2011-02-11 Markus Blaschke, TEQneers GmbH & Co. KG  <blaschke@teqneers.de>
	* Version 2.4.1
	* Fixed MySQL 5.0 "max-key-length"-issue (please update to MySQL 5.1 anyway...)
	* Renamed tx_tqseo_sitemap_pages to tx_tqseo_sitemap to make sure, that sitemap is cleaned up and table is up-2-date
	* Changed tx_tqseo_sitemap.page_hash to page_chash and implemented page_hash (md5) of url

2011-02-08 Markus Blaschke, TEQneers GmbH & Co. KG  <blaschke@teqneers.de>
	* Version 2.4.0
	* If you have problemes with updating tx_tqseo_sitemap_pages, remove table and recreate it with the extension manager, sorry...
	* Fixed empty link-url (Thanks to De Contardi Riccardo)
	* Cleanup (renamed tq_seo_cache to tx_tqseo_cache, feel free to remove tq_seo_cache)
	* Fixed possible duplicate urls in sitemap
	* Added support for page:lastUpdate (TYPO3 4.5, fallback to SYS_LASTCHANGED if empty)

ext_emconf.php
<?php

$EM_CONF[$_EXTKEY] = array(
	'title' => 'TEQneers SEO Enhancements',
	'description' => 'Search Engine Optimization (SEO), Indexed Google-Sitemap (TXT- and XML-Sitemap) for all Extensions (pibase, extbase), Metatags, Canonical-URL, Pagetitle manipulations, Crawler verification, Piwik and Google Analytics support and some more... multi-language- and multi-tree-support',
	'category' => 'misc',
	'shy' => 0,
	'version' => '6.0.0',
	'dependencies' => '',
	'conflicts' => '',
	'priority' => '',
	'loadOrder' => '',
	'module' => '',
	'state' => 'stable',
	'uploadfolder' => 1,
	'createDirs' => '',
	'modify_tables' => 'pages,pages_language_overlay',
	'clearcacheonload' => 1,
	'lockType' => '',
	'author' => 'Markus Blaschke',
	'author_email' => 'blaschke@teqneers.de',
	'author_company' => 'TEQneers GmbH & Co. KG',
	'CGLcompliance' => '',
	'CGLcompliance_note' => '',
	'constraints' => array(
		'depends' => array(
			'php' => '5.3.0-0.0.0',
			'typo3' => '6.1.0-6.2.99',
			'fluid' => '1.0.0-0.0.0',
		),
		'conflicts' => array(
		),
		'suggests' => array(
		),
	),
	'_md5_values_when_last_written' => 'a:31:{s:9:"ChangeLog";s:4:"95d7";s:10:"README.txt";s:4:"878d";s:16:"ext_autoload.php";s:4:"550a";s:21:"ext_conf_template.txt";s:4:"09c3";s:12:"ext_icon.gif";s:4:"6ce1";s:17:"ext_localconf.php";s:4:"4f36";s:14:"ext_tables.php";s:4:"6b22";s:14:"ext_tables.sql";s:4:"31cb";s:16:"locallang_db.xml";s:4:"a7ed";s:17:"locallang_tca.xml";s:4:"6623";s:7:"tca.php";s:4:"95ea";s:14:"doc/manual.pdf";s:4:"6b9f";s:14:"doc/manual.sxw";s:4:"0385";s:40:"hooks/sitemap/class.cache_controller.php";s:4:"b6d4";s:45:"hooks/sitemap/class.cache_controller_hook.php";s:4:"5b2d";s:27:"hooks/sitemap/locallang.xml";s:4:"0c9f";s:19:"lib/class.cache.php";s:4:"2659";s:18:"lib/class.http.php";s:4:"5366";s:24:"lib/class.linkparser.php";s:4:"a2e1";s:22:"lib/class.metatags.php";s:4:"0067";s:24:"lib/class.pagefooter.php";s:4:"35b6";s:23:"lib/class.pagetitle.php";s:4:"1709";s:24:"lib/class.robots_txt.php";s:4:"e839";s:19:"lib/class.tools.php";s:4:"b67d";s:34:"lib/sitemap/class.sitemap_base.php";s:4:"0e0a";s:37:"lib/sitemap/class.sitemap_indexer.php";s:4:"3162";s:33:"lib/sitemap/class.sitemap_txt.php";s:4:"fcc3";s:33:"lib/sitemap/class.sitemap_xml.php";s:4:"10bc";s:24:"res/ga-track-download.js";s:4:"e80d";s:28:"static/default/constants.txt";s:4:"60b5";s:24:"static/default/setup.txt";s:4:"e6ff";}',
	'suggests' => array(
	),
);

?>
